### PR TITLE
Add Artifact Hub repository metadata file

### PR DESCRIPTION
**Pull request by viceice**
# Pull Request

## Related issue
none

## What does this PR do?
- Add Artifact Hub repository metadata file to get the `Verified Publisher` badge at https://artifacthub.io/packages/helm/meilisearch-kubernetes/meilisearch

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by alallema** [comment]
Hi @viceice,
Thanks for this PR! Sorry for the delay, I was on vacation. I'll try to review this as soon as possible.

**Comment by alallema** [comment]
Hello @viceice,
I have thoroughly reviewed your PR and appreciate your addition of the repo to Artifact Hub is really nice 🎉  However, it would be ideal if the repo officially belonged to Meilisearch. Consequently, I'll be modifying the `repositoryID` and the email present in the artifacthub-repo.yml file.
By any chance, are you the owner of the Meilisearch organization name on the platform?
Thank a lot

**Comment by viceice** [comment]
yes, I'll happy to add one of the official maintainer and remove myself. 🤗

**Comment by viceice** [comment]
the repo id can stay the same, you can simply take over the entire organization

**Comment by viceice** [comment]
please make a suggestion for the owners, so I can add them to the organization as owners.

**Comment by alallema** [comment]
> please make a suggestion for the owners, so I can add them to the organization as owners.

You can put my email: amelie@meilisearch.com. Thanks 😊

**Comment by viceice** [comment]
> > please make a suggestion for the owners, so I can add them to the organization as owners.
> 
> You can put my email: amelie@meilisearch.com. Thanks 😊 

i need the artifacts hub username to invite you.

**Comment by alallema** [comment]
> 

Hi @viceice, it's my username is alallema

File: artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+# Artifact Hub repository metadata file
+repositoryID: '207118b4-2d63-47ea-8171-34aeb0aefba9'
+owners: # (optional, used to claim repository ownership)
+  - name: viceice
+    email: michael.kriese@visualon.de
```
Comment (alallema) [inline artifacthub-repo.yml:6]:
```suggestion
  - name: alallema
    email: amelie@meilisearch.com
```

**Review by alallema — COMMENTED** [review]
I received the invitation! Thank you. For security reasons, I will keep the Meilisearch organization for internal people only, and I changed the email address accordingly.
Thanks again for all your work on this topic 😊

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Comment by alallema** [comment]
bors merge

**Comment by alallema** [comment]
bors try

**Comment by alallema** [comment]
Hi @viceice,
I'm sorry, I just saw that you pushed your modification on the branch: `meilisearch:gh-pages` instead of `meilisearch:main`. Can I know why? I didn't find anything related to the documentation about it

**Comment by viceice** [comment]
yes, that file needs to be published together with the charts index.yaml file.

**Pull request merged**